python/lsst/ts/cbp/csc.py

```diff
@@ -69,7 +69,7 @@ def __init__(
         self.factor = factor
         self.log.info("CBP CSC initialized")
 
-    async def do_moveAzimuth(self, data):
+    async def do_move(self, data):
```

Comment:
Please update the doc string. Consider "Move the CBP mount to a specified position" or words to that effect.